BitwardenShared/Core/Vault/Services/TestHelpers/MockVaultClientService.swift

```diff
@@ -178,9 +178,16 @@ class MockClientCiphers: CiphersClientProtocol {
 
 class MockClientCollections: CollectionsClientProtocol {
     var decryptListError: Error?
+    
+    var decryptResult: (Collection) throws -> CollectionView = { collection in
+        CollectionView(collection: collection)
+    }
+    
+    var getCollectionTreeInput = [BitwardenSdk.CollectionView]()
+    var getCollectionTreeReturn: BitwardenSdk.CollectionViewTree?
```

Comment:
⛏️ Rename to `getCollectionTreeReceivedCollection` and `getCollectionTreeReturnValue` to align with auto generated mock format. Apply the same to the other service as well.